WebContent/js/reducers/jobNodes.js

```diff
@@ -123,6 +125,17 @@ export default function JobNodes(state = INITIAL_STATE, action) {
             });
         case STOP_REFRESH_ICON:
             return state.set('isFetching', false);
+        case RECEIVE_PURGE_JOB:
+            return state.merge({
+                jobs: state.get('jobs').remove(findKeyOfJob(state.get('jobs'), action.jobId)),
+            });
+        case RECEIVE_CANCEL_JOB: {
+            const jobs = state.get('jobs');
+            const jobKey = findKeyOfJob(state.get('jobs'), action.jobId);
```

Comment:
why not reference `jobs` one line above rather than doing `state.get('jobs')` twice? same goes for the RECEIVE_PURGE_JOB case above.

Reply:
Done